grammar.bnf

```diff
@@ -59,7 +59,14 @@ mident
   | ident
 
 iff
-  : '{{' 'if' expr '}}' expr '{{' 'else' '}}' expr '{{' 'end' '}}'
+  : '{{' 'if' expr '}}' expr elif_exprs '{{' 'else' '}}' expr '{{' 'end' '}}'
+
+elif_exprs
+  : {- empty -}
+  | elif_exprs elif_expr
+
+elif_exp
+  : '{{' 'elif' expr '}}' expr
```

Comment:
The grammar.bnf file is actually more out of date. Can you run this command from the root of the repo and commit the change?
```
happy src/Kriti/Parser/Grammar.y --pretty=grammar.bnf
```